package.json

```diff
@@ -50,5 +50,6 @@
 		"tailwindcss": "^3.4.4",
 		"typescript": "^5.5.3",
 		"typescript-eslint": "^7.16.0"
-	}
+	},
+	"packageManager": "pnpm@9.1.1+sha512.14e915759c11f77eac07faba4d019c193ec8637229e62ec99eefb7cf3c3b75c64447882b7c485142451ee3a6b408059cdfb7b7fa0341b975f12d0f7629c71195"
```

Comment:
tego nie powinno być, prawdopodobnie masz złą wersję pnpm'a

Reply:
hmm to jest deklaracja do `corepack`a

Reply:
ty masz pewnie zainstalowanego pnpma przez brew albo inaczej

Reply:
mogę to wywalić

Reply:
z tego co mi się wydaje to pojawia się to wtedy kiedy `pnpm-lock.yaml` został wygenerowany przez inną wersję pnpm niż jest zainstalowany, ale nie wiem. Kiedyś się z tym spotkałem i wystarczył update pnpm do odpowiedniej wersji i przestało się pojawiać, jeśli się mylę to popraw mnie.

Reply:
ale jaka jest poprawna wersja?

https://nodejs.org/api/packages.html#packagemanager

Reply:
ja mam najnowszą `pnpm@9.4.0` ale teraz patrzę i `pnpm-lock.yaml` ma w pierwszej linijce `lockfileVersion: '9.0'` 🤔